### PR TITLE
PR for uploading PP conf and SD in Asciidoc format and update the cPP module

### PR DESCRIPTION
**Pull request by n-kai**
I added two adoc files for PP configuration and SD. I also corrected some editorial errors in the cPP module.
Please simply approve this PR so that we can disscuss based on those latest version.

### CODE REVIEW

File: 3_ProtectionProfile/PP config.adoc [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+|[CEM] |Common Methodology for Information Technology Security Evaluation, +
+Evaluation Methodology, +
+CCMB-2017-04-004, Version 3.1 Revision 5, April 2017.
+|===
+
+*Protection Profiles*
+
+[cols=",",]
+|===
+|[MDFPP] |Protection Profile for Mobile Device Fundamentals, Version:3.1, 2017-06-16
```
Comment (The-Fiona) [inline 3_ProtectionProfile/PP config.adoc]: If it is the current 3.1 MDF then it cites CC V 3.1 R4.. It is the candidate MDF 3.2 that cites CC 3.1 R5

File: 3_ProtectionProfile/PP config.adoc [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+Note that to fully evaluate the TOE, these SARs shall be applied to the entire TSF and not just the portions described by [MDFPP] where the SARs are defined.
+
+In addition to this, both [MDFPP] and [BIOcPP Module] define "Evaluation Activities" for how to evaluate individual SFRs as they relate to the SARs for ASE_TSS.1, AGD_OPE.1, and ATE_IND.1. [MDFPP] and [BIOcPP Module] also provide Evaluation Activities for the SARs. In evaluating this PP-Configuration, the evaluator shall ensure that all Evaluation Activities for SFRs and SARs are evaluated as part of satisfying the required SARs.
+
+=== Related Documents
+
+**Common Criteria**footnote:[For details see http://www.commoncriteriaportal.org/]
+
+[cols=",",]
+|===
```
Comment (The-Fiona) [inline 3_ProtectionProfile/PP config.adoc]: We should cite the current addenda here.
Here I am assuming that we match the current draft of MDF (Candidate v 3.2) https://commoncriteria.github.io/pp/mobile-device/mobile-device-release.pdf

File: 3_ProtectionProfile/PP config.adoc [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+* Base-PP: Protection Profile for Mobile Device Fundamentals [MDFPP]
+* PP-Module: collaborative Protection Profile Module for Mobile biometric enrolment and verification - for unlocking the device - [BIOcPP Module].
+
+== Conformance Claims
+
+=== CC Conformance
+
+*Conformance Statement*
+
+To be conformant to this PP-Configuration, an ST must demonstrate Exact Conformance, a subset of Strict Conformance as defined in [CC1] (ASE_CCL).
```
Comment (The-Fiona) [inline 3_ProtectionProfile/PP config.adoc]: Sadly, neither [CC1} Not the addenda define the term "Exact Conformance".
Noting that many of the cPP "get around" this issue by defining it within the cPP.
The candidate MDF PP V3.2 is incorrect with the statement "An ST must claim exact conformance to this PP, as defined in the CC and CEM addenda for Exact Conformance, Selection-Based SFRs, and Optional SFRs (dated May 2017)." 

So I would suggest that this PP-Module avoid the issue for now and change the text to:
"To be conformant to this PP-Configuration, an ST must demonstrate Exact Conformance, a subset of Strict Conformance as defined in [MDFPP]."

File: 3_ProtectionProfile/PP config.adoc [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+| |ST Introduction (ASE_INT.1)
+| |Security Objectives for the Operational Environment (ASE_OBJ.1)
+| |Stated Security Requirements (ASE_REQ.1)
+| |Security Problem Definition (ASE_SPD.1)
+| |TOE Summary Specification (ASE_TSS.1)
+|Development (ADV) |Basic Functional Specification (ADV_FSP.1)
+|Guidance Documents (AGD) |Operational User Guidance (AGD_OPE.1)
+| |Preparative Procedures (AGD_PRE.1)
+|Life Cycle Support (ALC) |Labeling of the TOE (ALC_CMC.1)
+| |TOE CM Coverage (ALC_CMS.1)
```
Comment (The-Fiona) [inline 3_ProtectionProfile/PP config.adoc]: Add ALC_TSU_EXT

**Comment by The-Fiona** [comment]
Dear Naruki-san,
Sorry I am not very well familiar with the commenting/approving process :(
I did have some suggested changes (That I hope you can see.) But have approved the PR as you originally requested.

Does that work?

- Fiona

**Comment by n-kai** [comment]
@The-Fiona 
Thank you for your review and comment!
I will create another Pull Request to modify the draft as you suggested.
Naruki

**Comment by woodbe** [comment]
I have started a new PR #155 for updates on this which I expect to finish tomorrow (mainly small changes).

There are two points though that I would make:
1. For references in the document, wherever possible it should just use the <<...>> linking method instead of listing the "Section X" in the reference. I changed several of these because it will make editing easier over the long term since you wouldn't have to go back and edit all those references if the section numbers were changed due to larger document changes. There is another one on line 270 that I haven't yet figured out how to change the text for.
2. The beginning should be edited generally to better match the cPP. What I mean here is the ordering of the content. Things like the Glossary and Acronyms should be in the very first sections of the document as they are in the cPP. I wasn't ready for this more wholesale edit right now, but is something we should handle before the end of the month.

**Comment by woodbe** [comment]
@nils-tekampe I agree. Do you have specific suggestions based on the current text that you could put here?

File: 3_ProtectionProfile/PP config.adoc [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+|Tests (ATE) |Independent testing - conformance (ATE_IND.1)
+|Vulnerability Assessment (AVA) |Vulnerability Survey (AVA_VAN.1)
```
Comment (nils-tekampe) [inline 3_ProtectionProfile/PP config.adoc]: This line will have to be removed

**Comment by nils-tekampe** [comment]
Yes, at least kind of. First, we need to remove the line where the assurance component AVA_VAN is used. I just put a comment on this line. 
However, this is only the first step. As far as I understand it, it will not be possible to do this within a PP module while the whole PR currently makes the PP from a standalone PP into a module. 

So, my suggestion would be to 
1) remove AVA_VAN, 
2) go thru every change but only accept those ones that do not turn the PP into a module.

**Comment by woodbe** [comment]
Based on general agreement at this time (with @nils-tekampe not in favor) on the 3/14/19 call these changes will be accepted, including leaving AVA_VAN in the cPP-module.

There is still debate about whether this will be the final, and new issues may be opened to this question.

**Pull request merged**